src/util/ReactBatchedUpdates.js

```diff
@@ -0,0 +1,2 @@
+const {unstable_batchedUpdates} = require('ReactDOM');
```

Comment:
File needs standard copyright headers.  Rename to `Recoil_ReactBatchedUpdates.js`.